src/Providers/Gmail/Auth.php

```diff
@@ -93,6 +93,8 @@ public function get_client() {
 		// We request only the sending capability, as it's what we only need to do.
 		$client->setScopes( array( \Google_Service_Gmail::MAIL_GOOGLE_COM ) );
 		$client->setRedirectUri( self::get_plugin_auth_url() );
+		// Apply custom options to the client
```

Comment:
Please add a dot `.` here.
This is a WordPress PHP Code Style rule, that we follow in this project.

Thank you!